src/main/java/mate/academy/springintro/dto/user/UserResponseDto.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.springintro.dto.user;
+
+import lombok.Data;
+
+@Data
+public class UserResponseDto {
```

Comment:
example of response 
{
  "id": 1,
  "email": "john.doe@example.com",
  "firstName": "John",
  "lastName": "Doe",
  "shippingAddress": "123 Main St, City, Country"
}